examples/07_customers/03_get_customer_metafields/query.graphql

```diff
@@ -4,19 +4,18 @@
 
 # https://shopify.dev/tutorials/retrieve-metafields-with-storefront-api#expose-metafields-to-the-storefront-api
 
```

Comment:
Feel nitpicky again here but perhaps could make it a little more explicit that exposting metafields to the SFAPI it's a hard requirement  that you _must_ expose them?  Thoughts?

I only say this as Natalie and I had been doing [this](https://github.com/Shopify/storefront-api-learning-kit/blob/main/examples/05_collections/05_get_all_metafields_for_namespace_in_collection/query.graphql) on our Metafield related PR's.

Reply:
I modified the explanation of this query and referred "expose_metafield_to_SFAPI" query from metafields folder. Let me know if the new explanation makes sense to you folks :)